handbook/product/design/design_process.md

```diff
@@ -154,12 +154,23 @@ While asynchronous communication is a core attribute of remote work, key moments
 
 ### Metrics collection
 
-- In the RFC process, the definition of success will include items which can be measured and evaluated
-- Involve the [Business Operations & Strategy team](../../ops/bizops/index.md) in determining what metrics to track and how they are tracked
-- Work with engineers to add the required eventLog tracking
-- Tools
-    - Looker
-    - Google analytics
+- The definition of success for a project should be defined in each project's RFC or Product Document. This definition should include events which can be collected and measured to evaluate outcomes.
+- The locations and interaction moments where these metrics will be collected often must be defined within design files. 
+- To define a metric, designers should use a [metric card](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/%F0%9F%9B%A0Project-Tools?node-id=2597%3A6172) from the [project tools](https://www.figma.com/file/8qNcDzOXLj1hcOM76WDPN9/%F0%9F%9B%A0Project-Tools?node-id=0%3A1&viewport=-1298%2C3150%2C1) library. These cards help standardize the names and description of metrics across the product. 
+- A metric's name is defined in the following format:
+  - Project_Segment_INDEX-NUM_StageName
+    - *Project*: the name of the project. A project may have many metrics.
+    - *Segment*: the specific element (or series of elements) that are being measured. 
+    - *INDEX-NUM*: the events place in a funnel of related events, starting with 0. If the funnel has 3 stages, the last stage is 2. If a single event is being tracked this number will be 0
+    - *StageName*: a nice-name identifier for the metric’s position in the funnel
+  - Example: 
+    - A CTA funnel with 3 stages: CTA, a click on the CTA, and a user completing the sign up form would require the collection of at least 3 events:
+      - SignUpPLGMonitor_0_Tour
+      - SignUpPLGMonitor_1_CTA
+      - SignUpPLGMonitor_2_SignUpPage
```

Comment:
```suggestion
- Follow the [guidelines for naming events](../../ops/bizops/amplitude.md#adding-events-to-amplitude) when defining an event 
```

Reply:
I read through that page and the object action framework in Proper Case post from Segment. Makes sense. Would an example of a funnel event I've described above then be: SignUpPlgMonitor2SignUpPage, or better PlgMonitor2SignUp so it converts to: Plg Monitor 2 Sign Up?

If so could we document those funnel stages on that page?

Reply:
It'd be along the lines of:
- CodeMonitoringCTAViewed
- CodeMonitoringCTAClicked
- SignupPageViewed
- SignupCompleted

We'd be able to tie everything back to the specific campaign with the feature flag. cc @attfarhan though since he'll be owning more than I will going forward

Reply:
Love this new naming scheme. It'll help us remember what those metrics do exactly

Reply:
Ok, I'm 100% on board. Thanks for clarifying!!!